telemetry/ui/src/components/routes/app/DataView.tsx

```diff
@@ -328,6 +332,10 @@ const FormRenderer: React.FC<FormRendererProps> = ({ data, isDefaultExpanded: is
     return (
       <>
         {Object.entries(data).map(([key, value]) => {
+          if (value instanceof Array && value.length > 0 && typeof value[0] === 'number') {
+            // we want to display arrays of numbers as a single string.
+            value = value.toString();
```

Comment:
Fine for now but let's do `ReactJSON` (or whatever it is) for anything that wouldn't benefit from direct rendering